setup.py

```diff
@@ -22,9 +22,11 @@
     ),
     install_requires=[
         # when changing this list, also update docs/requirements.txt
-        "py-algorand-sdk>=1.9.0,<2.0.0",
-        "semantic-version>=2.9.0,<3.0.0",
         "docstring-parser==0.14.1",
+        "executing>=1.2.0",
```

Comment:
@tzaffi For _executing_ and _tabulate_ dependencies:  Unless you believe it's safe to allow a major version upgrade, can you specify upper bounds?

Reply:
Given the fragility of source mapping, this needs to be pinned to a precise version.